filestorage.go

```diff
@@ -311,15 +309,15 @@ func keepLockfileFresh(filename string) {
 		if err := recover(); err != nil {
 			buf := make([]byte, stackTraceBufferSize)
 			buf = buf[:runtime.Stack(buf, false)]
-			log.Printf("panic: active locking: %v\n%s", err, buf)
+			Default.Logger.Sugar().Panicf("active locking: %v\n%s", err, buf)
```

Comment:
I think we intentionally don't want to panic here but maybe I'm wrong. A panic would take down the server, we intentionally `recover()` to avoid that from happening, then just log that it happened and keep going.

Reply:
Yes, Panic should not be used here, I have replaced Panicf with Errorf.